### ISSUE TITLE
Save local image to cache after uploading image

### ISSUE THREAD

**Issue by coolcool1994** [bumptech/glide#4304]
After I upload an image to Firebase, I want to cache that local image (Bitmap, byte array, or file) into Glide so that Glide doesn't have to fetch the image that I just uploaded to Firebase because it cost money to download images from Firebase.

I asked so many questions and searched for so many posts, but no answer. Does Glide not have this simple feature to save local image to cache? PLEASE HELPPPPPP

**Comment by Hotdogcode** [comment]
Apply DiskCacheStrategy :- 

`Glide.with(fragment)
  .load(url)
  .diskCacheStrategy(DiskCacheStrategy.ALL)
  .into(imageView);`

Load From Cache : -

`Glide.with(fragment)
  .load(url)
  .onlyRetrieveFromCache(true)
  .into(imageView);`

**Comment by coolcool1994** [comment]
I don’t think you understood the question. I do not want to make the network call at all. In fact I am trying to prevent network call. Why would you do Glide.with(fragment) .load(url) .diskCacheStrategy(DiskCacheStrategy.ALL) .into(imageView); to the image that you just uploaded? This would download the image that I uploaded to the server which can be preventable because I just uploaded the image and has a local data for it. My question is how to put local image into cache to a url so that future Glide fetch to that url returns that image from cache.

I don’t think this is possible in Glide. Such a simple, needed feature though. In iOS, SDWebInage you can do this with one line of code.
